features/upgrade/node/node-upgrade.feature

```diff
@@ -45,10 +45,10 @@ Feature: Node components upgrade tests
     When I run commands on the host:
       | cat /etc/kubernetes/kubelet.conf |
     Then the step should succeed
-    And the output should contain:
-      | "imageMinimumGCAge": "5m0s"       |
-      | "imageGCHighThresholdPercent": 80 |
-      | "maxPods": 240                    |
+    And the output should match:
+      | [\"]?imageMinimumGCAge[\"]?: [\"]?5m0s[\"]? |
```

Comment:
`[\"]?` can be shorten as `"?`

Reply:
I think [\"]? is more accurate than ? , means only allow appearing " but not other charater

Reply:
I think I posted it as `"?`, not `?`